dev/cnf/oss_dependencies.maven

```diff
@@ -602,7 +602,9 @@ org.rnorth.duct-tape:duct-tape:1.0.8
 org.rnorth.visible-assertions:visible-assertions:2.1.2
 org.seleniumhq.webdriver:webdriver-common:0.9.7376
 org.slf4j:slf4j-api:1.7.30
+org.slf4j:slf4j-api:1.7.32
 org.slf4j:slf4j-simple:1.7.30
+org.slf4j:slf4j-simple:1.7.32
```

Comment:
please don't duplicate dependency versions ... update the existing one and move up to the latest version available

Reply:
latest version is 1.7.36